syntax = "proto3";

package order;

service OrderService {
  rpc CreateOrder (OrderRequest) returns (OrderConfirmation) {}
}

message OrderRequest {
  string customer_name = 1;
  string product_name = 2;
  int32 quantity = 3;
}

message OrderConfirmation {
  string order_id = 1;
  string message = 2;
}